### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Andjelaaaa**
Add additional resource on how to connect to the brainhack cloud from discussion on issue #243 

Changes proposed in this pull request:

- add link to external repo with detailed guidance

### CODE REVIEW

**Review by clarkenj — COMMENTED** [review]
Looks great but I'm wondering if we should just link to https://github.com/brainhack-school2022/dimitrijevic_project/blob/main/BrainHackCloud_steps/get_connected.md - are the other files relevant?

**Comment by Andjelaaaa** [comment]
> Looks great but I'm wondering if we should just link to https://github.com/brainhack-school2022/dimitrijevic_project/blob/main/BrainHackCloud_steps/get_connected.md - are the other files relevant?

Yes it might be better to link it directly to that file as the other one gives instructions on connecting to their Neurodesktop container with various neuroimaging tools. For this specific HPC module, it might not be relevant!

File: content/en/modules/HPC/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ For example, the documentation website for the clusters is no longer https://doc
 
 `ssh <username>@beluga.computecanada.ca`
 
-:warning: :warning: If you reside outside of Canada and don't have access to a local HPC you can apply to access [Brainhack Cloud](https://brainhack.org/brainhack_cloud/). Subsequently anytime the lecture material below refers to location-specific inputs you will have to adjust your inputs accordingly (e.g. the ssh connection).
+:warning: :warning: If you reside outside of Canada and don't have access to a local HPC you can apply to access [Brainhack Cloud](https://brainhack.org/brainhack_cloud/). Some detailed steps for connecting to their cloud are available on this [GitHub repository](https://github.com/brainhack-school2022/dimitrijevic_project/tree/main/BrainHackCloud_steps). Subsequently anytime the lecture material below refers to location-specific inputs you will have to adjust your inputs accordingly (e.g. the ssh connection).
```
Comment (Andjelaaaa) [inline content/en/modules/HPC/index.md:47]: Link updated to a more relevant page
```suggestion
:warning: :warning: If you reside outside of Canada and don't have access to a local HPC you can apply to access [Brainhack Cloud](https://brainhack.org/brainhack_cloud/). Some detailed steps for connecting to their cloud are available on this [GitHub repository](https://github.com/brainhack-school2022/dimitrijevic_project/blob/main/BrainHackCloud_steps/get_connected.md). Subsequently anytime the lecture material below refers to location-specific inputs you will have to adjust your inputs accordingly (e.g. the ssh connection).
```

**Review by clarkenj — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**